src/NLog/LayoutRenderers/Wrappers/XmlEncodeLayoutRendererWrapper.cs

```diff
@@ -57,6 +57,17 @@ public sealed class XmlEncodeLayoutRendererWrapper : WrapperLayoutRendererBase
         /// <docgen category="Layout Options" order="10"/>
         public bool XmlEncode { get; set; } = true;
 
+        /// <summary>
+        /// Indicates whether the rendered value should be wrapped in <![CDATA[ ... ]]> section.
+        /// </summary>
+        public bool WrapInCData { get; set; } = false;
+
+        /// <summary>
+        /// Remove Imvalid Xml Characters from inner
+        /// </summary>
+        public bool RemoveInvalidXml { get; set; } = true;
```

Comment:
Can this be merged together with `XmlEncode`-property? Only when XmlEncode = false, then it should be raw-append without validation.